rootfs-builder/README.md

```diff
@@ -0,0 +1,57 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+Kata Containers guest OS is created using `rootfs.sh`.
+
+## Supported base OS ##
+
+The `rootfs.sh` script support different based OS `rootfs`. To build
+a `rootfs` based in a supported based OS run:
+
+``` ./rootfs.sh <distro> ```
```

Comment:
Can you format this across 3 lines as it's rendering differently to other fixed-font chunks.